dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/dex/DexApiSyntax.scala

```diff
@@ -23,7 +24,23 @@ object DexApiSyntax {
       waitForOrderStatus(order.assetPair, order.id(), status)
 
     def waitForOrderStatus(assetPair: AssetPair, id: Order.Id, status: HttpOrderStatus.Status): F[HttpOrderStatus] =
-      waitForOrder(assetPair, id)(_.status == status)
+      waitForOrder(assetPair, id) { s =>
+
+        def fail() = throw new RuntimeException(s"Expected status $status for order $id, but found ${s.status}")
+        def check(): Boolean = s.status == status
+
+        status match {
```

Comment:
Let's rename `status` to `expectedStatus` to distinguish `s.status` and `status`.

Reply:
ok